planner/solver.go

```diff
@@ -493,8 +499,18 @@ func (s *Solver) generatePlacesForSlots(ctx context.Context, req *PlanningReques
 		}
 		logger.Debugf("Before filtering, the number of places for category %s is %d", slot.Category, len(places))
 
-		placesByTime, err := s.TimeMatcher.Match(&matching.FilterRequest{
+		placesByRating, err := s.UserRatingMatcher.Match(&matching.FilterRequest{
```

Comment:
We should keep the time matcher...

Reply:
the time matcher is still here, just a few lines down at line 512.